File: LLM-Translations/GPT-4/codenet/Java/Python/s125881159.py
import sys

def main():
    n = int(input().strip())
    d = 0
    nums = list(map(int, input().strip().split()))
    for num in nums:
        d += num % 2
    print("YES" if d % 2 == 0 else "NO")

if __name__ == "__main__":
    main()

File: LLM-Translations/GPT-4/codenet/Java/Go/s125881159.go
package main

import (
	"fmt"
	"os"
	"bufio"
	"strconv"
	"strings"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	n, _ := strconv.Atoi(scanner.Text())
	d := 0
	scanner.Scan()
	inputs := strings.Split(scanner.Text(), " ")

	for i := 0; i < n; i++ {
		num, _ := strconv.Atoi(inputs[i])
		d += num % 2
	}

	if d%2 == 0 {
		fmt.Println("YES")
	} else {
		fmt.Println("NO")
	}
}
// 
